modules/common/test/helpers/networkattachmentdefinition.go

```diff
@@ -36,7 +36,7 @@ func (tc *TestHelper) CreateNetworkAttachmentDefinition(name types.NamespacedNam
 			Namespace: name.Namespace,
 		},
 		Spec: networkv1.NetworkAttachmentDefinitionSpec{
-			Config: "",
+			Config: `{}`,
```

Comment:
empty string isn't a valid json object I assume?

Reply:
yes, unmarshal it as part of https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/networkattachment/networkattachment.go#L158 would fail with:
```
  2024-11-13T15:33:24+01:00     ERROR   Reconciler error        {"controller": "cinder", "controllerGroup": "cinder.openstack.org", "controllerKind": "Cinder", "Cind
er": {"name":"cinder-6af4b","namespace":"4eed3b56-5a11-486a-a472-00a9499f9289"}, "namespace": "4eed3b56-5a11-486a-a472-00a9499f9289", "name": "cinder-6af4b", "reconcileID": "526ff203-812e-4e6e-9608-7f1a28b8084b", "error": "failed create network annotation from [internalapi]: failed to unmarshal JSON data: unexpected end of JSON input"} 
```